e2e/test/scenarios/binning/binning-options.cy.spec.js

```diff
@@ -80,6 +80,9 @@ const LONGITUDE_BUCKETS = [
   "Bin every 1 degree",
   "Bin every 10 degrees",
   "Bin every 20 degrees",
+  "Bin every 0.05 degrees",
```

Comment:
Should we format the values according to "[Separator style](https://www.metabase.com/docs/latest/data-modeling/formatting#numbers)" setting?